ci/ciimage/ubuntu-rolling/install.sh

```diff
@@ -50,6 +51,18 @@ dub_fetch dubtestproject
 dub build dubtestproject:test1 --compiler=ldc2
 dub build dubtestproject:test2 --compiler=ldc2
 
+# zig
+
+ZIG_ARCHIVE=$(wget -qO- https://ziglang.org/download/index.json | jq -r ".master[\"$(uname -m)-linux\"].tarball")
```

Comment:
I'm not sure if we want to pull master for this. That'll lead to flaky CI. If we do want that, it needs justification.

Reply:
Yeah, we can probably use the latest release. I just need to figure out the jq to read the 2nd object.

Reply:
Alright, we're now using a tagged release instead of master.